### PR TITLE
fix: Fix filters and search for my reports page [4]

### PR DESCRIPTION
**Pull request by bistaastha**
Test fixes in follow up

### CODE REVIEW

File: src/app/fyle/my-reports/my-reports.page.ts [hunk 1]
```diff
@@ -453,9 +454,10 @@ export class MyReportsPage {
   }> {
     const currentParams = this.loadData$.getValue();
     currentParams.pageNumber = 1;
-    const newQueryParams: { or: string[]; and?: string } = {
-      or: [],
-    };
+    const newQueryParams: {
```
Comment (bistaastha) [inline src/app/fyle/my-reports/my-reports.page.ts:457]: Usage specific to this page

**Review by mvaishnavi — APPROVED** [review]

**Review by suyashpatil78 — APPROVED** [review]

File: src/app/fyle/my-reports/my-reports.page.ts [hunk 2]
```diff
@@ -386,43 +387,43 @@ export class MyReportsPage {
...
-        stateOrFilter.push('state.in.(PAID)');
+        stateOrFilter.push('PAID');
       }
     }
 
     if (stateOrFilter.length > 0) {
-      let combinedStateOrFilter = stateOrFilter.reduce((param1, param2) => `${param1}, ${param2}`);
-      combinedStateOrFilter = `(${combinedStateOrFilter})`;
-      newQueryParams.or.push(combinedStateOrFilter);
+      let combinedStateOrFilter = stateOrFilter.reduce((param1, param2) => `"${param1}","${param2}"`);
```
Comment (Chethan-Fyle) [inline src/app/fyle/my-reports/my-reports.page.ts:424]: Can't we use `stateOrFilter.join(',')`?

**Review by Chethan-Fyle — COMMENTED** [review]
One comment.

File: src/app/fyle/my-reports/my-reports.page.ts (hunk 2 above)
Comment (bistaastha) [reply src/app/fyle/my-reports/my-reports.page.ts:424]: fixed in follow up

**Review by Chethan-Fyle — APPROVED** [review]

**Pull request closed without merging**